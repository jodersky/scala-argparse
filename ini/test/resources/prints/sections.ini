a=1

[section1]
a.b=1

[section2]
a=1
b=2

[section1.b]
c=1

[section1.a]
c=1

[section1]
c=1
