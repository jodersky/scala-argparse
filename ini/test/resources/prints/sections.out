a=1

[section1]
  c=1

[section1.a]
  b=1
  c=1

[section1.b]
  c=1

[section2]
  a=1
  b=2
